index.html

```diff
@@ -11,6 +11,7 @@
       #messages { list-style-type: none; margin: 0; padding: 0; }
       #messages li { padding: 5px 10px; }
       #messages li:nth-child(odd) { background: #eee; }
+      #msg { margin-bottom: 40px }
```

Comment:
This one doesn't work for me in Firefox nor Safari - the lowest two lines are still covered by the chat block.